tests/integration/test_content_manifest.py

```diff
@@ -31,12 +31,13 @@ def test_valid_content_manifest_request(test_env, default_requests):
     """
     client = utils.Client(test_env["api_url"], test_env["api_auth_type"], test_env.get("timeout"))
 
-    initial_response = default_requests["gomod"].initial_response
-    content_manifest_response = client.fetch_content_manifest(initial_response.id)
-    assert content_manifest_response.status == 200
+    for default_request in default_requests.values():
```

Comment:
Should we make it configurable which package managers should be tested here? That way, if we add support for a package manager but do not have the content manifest generation implemented yet, we can specify in configuration that it should not be tested.

Reply:
Now that I think about it, would it even fail? Or would the generated manifest just be empty?

Reply:
> Should we make it configurable which package managers should be tested here? 
Done. I added `pkg_managers: ["gomod", "npm"]` into `test_env_vars.yaml`.

Reply:
> Now that I think about it, would it even fail? Or would the generated manifest just be empty?

it will fail in cases:
- cachito will not be able to download dependencies, 
- `content_manifest_schema` will not be valid

Reply:
@chmeliik ^^ :)